.github/workflows/ci.yml

```diff
@@ -197,3 +197,28 @@ jobs:
         typo3-version:
           - ^9.5
           - ^10.4
+  security-test:
+    name: "PHP Security test"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: "Install PHP"
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: "${{ matrix.php-version }}"
+      - name: "Cache dependencies installed with composer"
+        uses: actions/cache@v1
+        with:
+          key: "php${{ matrix.php-version }}
+               -composer-${{ hashFiles('**/composer.json') }}"
+          path: ~/.composer/cache
+          restore-keys: "php${{ matrix.php-version }}-composer-\n"
+      - name: "Install Composer dependencies"
+        run: "composer install --no-progress"
+      - name: "Run PHP Security test"
+        run: "composer ci:tests:security"
+    strategy:
+      matrix:
+        php-version:
+          - 7.4
```

Comment:
Please let's move the matrix up so the reader has seen it when they see the matrix variable getting used.

Reply:
You mean before "steps"? If so, then we should also move this for other jobs

Reply:
Yes, before "steps". For the existing jobs, please let's do this in a separate PR so we don't mix things up.